lib/internal/errors.js

```diff
@@ -707,6 +707,15 @@ E('ERR_INVALID_PERFORMANCE_MARK', 'The "%s" performance mark has not been set');
 E('ERR_INVALID_PROTOCOL', 'Protocol "%s" not supported. Expected "%s"');
 E('ERR_INVALID_REPL_EVAL_CONFIG',
   'Cannot specify both "breakEvalOnSigint" and "eval" for REPL');
+E('ERR_INVALID_SIGNATURE', (fn, signature, args) => {
+  const formatted = lazyUtil().inspect(args, {
+    depth: 0,
+    breakLength: Infinity
+  }).replace(/^\[\s*/, '(').replace(/\s*\]$/, ')');
+  return 'Function was invoked with an unknown combination of arguments.' +
+         `\nExpected ${fn}${signature}\nReceived ${fn}${formatted}`;
```

Comment:
What do you think about adding a extra new line before `Expected` and at the end of the message and use colons after `Expected` and `Received`?

Reply:
Colons SGTM, not sure about the new lines...do we have other types of errors that put empty linebreaks?

Reply:
Simple assert does that. I personally feel like adding empty lines is good in case our errors are longer than a single line.